.gitpod.Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM gitpod/workspace-full
 
-RUN wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
-    sudo dpkg -i packages-microsoft-prod.deb && \
-    rm packages-microsoft-prod.deb;
-
-RUN sudo apt-get update; \
-  sudo apt-get install -y apt-transport-https && \
-  sudo apt-get update && \
-  sudo apt-get install -y dotnet-sdk-6.0;
-
-RUN sudo apt-get update; \
-  sudo apt-get install -y apt-transport-https && \
-  sudo apt-get update && \
-  sudo apt-get install -y dotnet-sdk-6.0
\ No newline at end of file
+USER root
```

Comment:
Note: We use `USER root` to get root permission, so that we can remove `sudo` below. cc @filiptronicek